scripts/router-dashboard-fail2ban-snapshot.py

```
@@ -0,0 +1,100 @@
+import grp
+import json
+import os
+import subprocess
+import tempfile
+
+FAIL2BAN_CLIENT = os.environ["DASHBOARD_FAIL2BAN_CLIENT"]
+OUTPUT_PATH = os.environ["DASHBOARD_FAIL2BAN_STATUS_FILE"]
+
+
+def build_payload():
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)